<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>750</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>PhoneBook</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>751</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>900</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>1000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(60, 60, 60);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>49</y>
       <width>661</width>
       <height>541</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:8px;
background-color: rgb(85, 170, 127);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableWidget" name="phone_table">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>6</y>
        <width>649</width>
        <height>529</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>649</width>
        <height>529</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>649</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius:9px;
</string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>5</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>5</number>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>NAME</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="background">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>LAST NAME</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DDD</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PHONE</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>211</width>
       <height>731</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:8px;
background-color: rgb(25,25,25);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>10</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>PhoneBook</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="add_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>492</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>FILL THE FIELDS BELOW THEN CLICK ME</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
	background-color: rgb(0, 85, 0);
	border:2px solid rgb(60,60,60);
	border-radius:8px;
	color: rgb(255,255,255);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
	border-radius:10px
}
QPushButton:pressed {	
	background-color: rgb(231,52,19);
	border:2px solid rgb(201,52,19);
	color: rgb(200,200,200);
	border-radius:12px
}</string>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
     </widget>
     <widget class="QPushButton" name="del_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>541</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>SELECT REGISTER TO DELETE THEN CLICK ME</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
	background-color: rgb(0, 85, 0);
	border:2px solid rgb(60,60,60);
	border-radius:8px;
	color: rgb(255,255,255);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
	border-radius:10px
}
QPushButton:pressed {	
	background-color: rgb(231,52,19);
	border:2px solid rgb(201,52,19);
	color: rgb(200,200,200);
	border-radius:12px
}</string>
      </property>
      <property name="text">
       <string>DELETE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="search_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>383</y>
        <width>191</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
	background-color: rgb(0, 85, 0);
	border:2px solid rgb(60,60,60);
	border-radius:8px;
	color: rgb(255,255,255);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
	border-radius:10px
}
QPushButton:pressed {	
	background-color: rgb(231,52,19);
	border:2px solid rgb(201,52,19);
	color: rgb(200,200,200);
	border-radius:12px
}</string>
      </property>
      <property name="text">
       <string>SEARCH</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>62</y>
        <width>191</width>
        <height>73</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(115, 115,115);
border-radius:8px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>66</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid rgb(60,60,60);
color: rgb(255, 255, 255);
background-color: rgb(0, 85, 0);</string>
      </property>
      <property name="text">
       <string>RECORDS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="records_numbers">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>101</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px solid rgb(60,60,60);
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="help_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>680</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>SELECT REGISTER TO DELETE THEN CLICK ME</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {	
	background-color: rgb(0, 85, 0);
	border:2px solid rgb(60,60,60);
	border-radius:8px;
	color: rgb(255,255,255);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
	border-radius:10px
}
QPushButton:pressed {	
	background-color: rgb(231,52,19);
	border:2px solid rgb(201,52,19);
	color: rgb(200,200,200);
	border-radius:12px
}</string>
      </property>
      <property name="text">
       <string>HELP</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="output">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>9</y>
       <width>621</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>590</y>
      <width>676</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(60,60,60);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="campo_name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius:8px;
</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="campo_lastname">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius:8px;
</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="campo_phone">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius:8px;
</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="campo_ddd">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius:8px;
</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>15</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>NAME</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>85</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>LAST NAME</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>15</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>DDD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>15</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>PHONE NUMBER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ok_btn">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>110</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-color: rgb(0, 85, 0);
	border:2px solid rgb(40,40,40);
	border-radius:8px;
	color: rgb(255,255,255);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
	border-radius:10px
}
QPushButton:pressed {	
	background-color: rgb(231,52,19);
	border:2px solid rgb(201,52,19);
	color: rgb(200,200,200);
	border-radius:12px
}</string>
     </property>
     <property name="text">
      <string>CONFIRM EDIT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="edit_btn">
     <property name="geometry">
      <rect>
       <x>299</x>
       <y>110</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>SELECT REGISTER TO EDIT THEN CLICK ME</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-color: rgb(0, 85, 0);
	border:2px solid rgb(40,40,40);
	border-radius:8px;
	color: rgb(255,255,255);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
	border-radius:10px
}
QPushButton:pressed {	
	background-color: rgb(231,52,19);
	border:2px solid rgb(201,52,19);
	color: rgb(200,200,200);
	border-radius:12px
}</string>
     </property>
     <property name="text">
      <string>EDIT</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
